apps/vaporgui/VaporTable.cpp

```diff
@@ -98,6 +99,14 @@ void VaporTable::Update(int rows, int cols,
 	resizeTableHeight();
 }
 
+void VaporTable::SetVerticalHeaderWidth(int width) {
+    _verticalHeaderWidth = width;
```

Comment:
Seems like {Set,Get}VerticalHeaderWidth could be pure wrapper functions, no? I.e. call _table->{set,get}VerticalHeaderWidth directly?